aspnetcore/client-side/spa/blazor/index.md

```diff
@@ -50,7 +50,7 @@ Blazor supports core facilities required by most apps, including:
 
 To reduce the size of the downloaded app unused code stripped out of the app when it's published by the [Intermediate Language (IL) Linker](xref:host-and-deploy/razor-components/configure-linker).
 
-Blazor is the client-side hosting model for Razor Components. Because Razor Components decouple a component's rendering logic from how UI updates are applied, there's flexibility in how Razor Components can be hosted. Use ASP.NET Core Razor Components to host Razor Components on the server in an ASP.NET Core app where UI updates are handled over a SignalR connection. For more information, see <xref:razor-components/hosting-models#server-side-hosting-model>. 
+Blazor is the client-side hosting model for Razor Components. Because Razor Components decouple a component's rendering logic from how UI updates are applied, there's flexibility in how Razor Components can be hosted. Use ASP.NET Core Razor Components to host Razor Components on the server in an ASP.NET Core app where UI updates are handled over a SignalR connection. For more information, see <xref:razor-components/index> and <xref:razor-components/hosting-models#server-side-hosting-model>. 
```

Comment:
consider making Razor Components a link and ditto for ASP.NET Core Razor Components

Reply:
I generally don't when the paragraph will end with our "For more info ..." bit. Otherwise, the reader might be inclined to click early (first appearance). I'd prefer the reader click at the end of the paragraph (For more info ...) after they've read it. Up to you tho ... I need to know if you want it as a blanket rule, since I don't normally ever set links that way.

Reply:
The other thing in this case is order: I don't want the reader going to hosting models before going to the intro topic. By listing the links in the order that I want them to click, I have a better chance of steering them to content in an orderly way (general to specific).

Reply:
Does the link differentiate between the two?

Reply:
Yes ... The first one will show up **Introduction to Razor Components** and the second one will say **Razor Components hosting models**. That's the best order for them to visit (if they do).

Reply:
The other topic will show the links as ...

* **Introduction to Blazor**
* **Razor Components hosting models**